### ISSUE TITLE
Windows always appear on first screen

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#37320]
**Godot version:**
5f1107aa3

**OS/device including version:**
Windows 10

**Issue description:**
Any new Window will appear on the first screen (monitor). IMO they should appear on the same screen as parent Window.

**Steps to reproduce:**
1. Open Godot editor
2. Move it to second screen
3. Add node
4. Dialog appears on another screen :/

EDIT:
I just noticed that any popup (including sub-menus) have this issue ;_;

**Comment by Calinou** [comment]
I can confirm this on commit 307b1b3a5 and fafb0ba06.

**Comment by KoBeWi** [comment]
For some reason the popups are working correctly in 163687d, but widows are still broken (settings etc.)

**Comment by HaSa1002** [comment]
> 
> 
> For some reason the popups are working correctly in [163687d](https://github.com/godotengine/godot/commit/163687d17a8a11da3cf1a3595c511a5f8fc94571), but widows are still broken (settings etc.)

Yes but it will break as soon ass you move your move window onto another screen. This was improved in https://github.com/godotengine/godot/commit/08e80ccd999bf8bf2c80d2300097ceb82e3f8c42

**Comment by akien-mga** [comment]
This is not fixed by #41395 which is specific to tooltips.

**Comment by KoBeWi** [comment]
I don't remember if it was the case originally, but if you move the window to the second screen, it will then appear there.
Still, they should always appear at the same screen as the main window.

**Comment by KoBeWi** [comment]
Actually it's fine now 🤔
The dialogs will remember their position, including the screen. If you don't move editor to another screen after the dialog was open then it will appear correctly.

It's not a general issue anymore, we could solve it per-dialog. Although we don't get complains about it.
